pettingzoo/test/api_test.py

```diff
@@ -331,12 +328,11 @@ def play_test(env, observation_0, num_cycles):
             env.infos[agent], dict
         ), "an environment agent's info must be a dictionary"
         prev_observe, reward, terminated, truncated, info = env.last()
-        if terminated or truncated:
-            action = None
-        elif isinstance(prev_observe, dict) and "action_mask" in prev_observe:
-            action = random.choice(np.flatnonzero(prev_observe["action_mask"]).tolist())
+        if isinstance(prev_observe, dict) and "action_mask" in prev_observe:
+            mask = prev_observe["action_mask"]
         else:
-            action = env.action_space(agent).sample()
+            mask = None
+        action = env.action_space(agent).sample(mask)
```

Comment:
Are we concerned that mask may not be None even when the env has terminated/truncated?